uaclient/api/u/pro/attach/magic/initiate/v1.py

```diff
@@ -21,25 +22,29 @@ class MagicAttachInitiateResult(DataObject, AdditionalInfo):
 
     def __init__(
         self,
-        _schema: str,
         user_code: str,
         token: str,
         expires: str,
         expires_in: int,
     ):
-        self._schema = _schema
         self.user_code = user_code
         self.token = token
         self.expires = expires
         self.expires_in = expires_in
 
 
-def initiate(cfg: UAConfig) -> MagicAttachInitiateResult:
+def initiate() -> MagicAttachInitiateResult:
+    return _initiate()
+
+
+def _initiate(*, cfg: Optional[UAConfig] = None) -> MagicAttachInitiateResult:
+    if cfg is None:
+        cfg = UAConfig()
```

Comment:
I like this approach that covers both sides of using the API. I have no problems going forward with this refactoring